providers/Tidal/v2/lookup.ts

```diff
@@ -0,0 +1,278 @@
+import { join } from 'std/url/join.ts';
+import { ReleaseApiLookup } from '@/providers/base.ts';
+import TidalProvider from '@/providers/Tidal/mod.ts';
+import {
+	AlbumItemResourceIdentifier,
+	AlbumsResource,
+	MultiDataDocument,
+	ProvidersResource,
+	SingleDataDocument,
+	TracksResource,
+} from '@/providers/Tidal/v2/api_types.ts';
```

Comment:
Some of the automatically generated import statements further down have not been combined with the existing one for this module, for whatever reason. Can you please clean these up manually?

Using `import type` whenever possible also prevents the dev server from reloading when a type-only module gets changed.
```suggestion
import type {
	AlbumItemResourceIdentifier,
	AlbumsResource,
	MultiDataDocument,
	ProvidersResource,
	SingleDataDocument,
	TracksResource,
} from '@/providers/Tidal/v2/api_types.ts';
```
Regarding the sort order of import statements I still haven't made a decision, so I won't comment on that for now 😅

Reply:
Done in latest rebase